.github/workflows/playground_examples_reusable.yml

```diff
@@ -27,14 +27,73 @@ on:
       subdirs:
         type: string
         required: true
+      whitelist:
```

Comment:
Nit: Can we use `allowlist` instead of `whitelist`? There's a general industry move towards allowlist for [clarity](https://www.linkedin.com/pulse/allowlist-blocklist-better-terms-everyone-lets-use-them-rob-black/) and [inclusiveness](https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white).